human_experiments/scripts/start_lab_recorder

```diff
@@ -93,11 +93,11 @@ echo " "
 # This is done so that git will not detect on any branch that the config files have been modified.
 echo -e "Restoring the 3 LabRecorder config files back to what is in the main git repository on CAT:"
 # shellcheck disable=SC2029
-ssh "$USER_ON_TARGET@$TARGET_IP" "cd ${CONFIG_DIR} && git checkout LabRecorder_Lion_22345.cfg"
+#ssh "$USER_ON_TARGET@$TARGET_IP" "cd ${CONFIG_DIR} && git checkout LabRecorder_Lion_22345.cfg"
```

Comment:
I believe the lines should be uncommented. We commented them because, at the time, we were doing something on CAT, and we did not want this to override the changes. @rchamplin, can you confirm that all your modifications to the fNIRS config files are already in the repo? If so, we can uncomment the lines so that the fNIRS config files on CAT will return to their original state after being loaded with the details of the current experiment.

Reply:
Okay